web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -117,24 +93,23 @@ BmcMangaList.prototype.generate = function() {
     }
 
     // Now that the parent is a clean slate, let's generate
-    // bookmarks.forEach(bkmk => {
-    //     comic = new BmcComic(bkmk.label, bkmk.id, 3, 21);
-    //     comic.addSource(new BmcComicSource(bkmk.label, 'mangaeden'));
-    //     comic.addSource(new BmcComicSource(bkmk.label, 'mangafox'));
-    //     comic.addSource(new BmcComicSource(bkmk.label, 'mangahere'));
-    //     comic.addSource(new BmcComicSource(bkmk.label, 'mangareader'));
-    //     console.warn('fake comic generated for', bkmk.label);
-    bmcEngine._db.list((err, comics) => {
+    bmcDb.list((err, comics) => {
        comics.forEach(comic =>
             mangaList.appendChild(this.generateComic(comic))
        );
     });
 }
 
 BmcMangaList.prototype.setMode = function(mode) {
+    var btn = document.querySelector('.button-add');
     switch (mode) {
```

Comment:
I have a small preference over `if else` construction but that's not an issue in itself. :)

Reply:
I'm the opposite :D